app/routes.go

```diff
@@ -1,23 +1,34 @@
 package app
 
 import (
+	"fmt"
 	"net/http"
 
 	"github.com/go-chi/chi/v5"
 	"github.com/go-chi/chi/v5/middleware"
 )
 
 func loadRoutes() *chi.Mux {
-	router := chi.NewRouter();
+	router := chi.NewRouter()
 
-	router.Use(middleware.Logger);
-	router.Get("/", func (w http.ResponseWriter, r *http.Request) {
+	router.Use(middleware.Logger)
+	router.Get("/api/v2/mcap", func(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusOK)
+
+		queryParams := r.URL.Query()
+		date := queryParams.Get("date")
+		location := queryParams.Get("location")
+		notes := queryParams.Get("notes")
+		eventType := queryParams.Get("eventType")
+
+		response := fmt.Sprintf("date: %s, location: %s, notes: %s, eventType: %s", date, location, notes, eventType)
```

Comment:
queryParms.get returns a map. You should definitely add checks whether or not the given parameters are present or not, so you don't run into any nil issues.